base/servicehub/hub.go

```diff
@@ -312,7 +312,8 @@ func (h *Hub) Start(closer ...<-chan os.Signal) (err error) {
 	for _, ch := range closer {
 		go func(ch <-chan os.Signal) {
 			select {
-			case <-ch:
+			case signal := <-ch:
+				fmt.Printf("received signal: %s\n", signal.String())
```

Comment:
should print more readable like `hub quitting ... received ...`

Reply:
adjusted.
<img width="678" alt="image" src="https://user-images.githubusercontent.com/13919034/160373198-681cf0ff-4a8a-4bf6-be46-b252ba6770de.png">